frontend/src/components/Header/components/LeftSection.tsx

```diff
@@ -0,0 +1,21 @@
+import logoSmall from "@/assets/imgs/logoSmall.png";
+
+interface HeaderProps {
+  lecture?: boolean;
+  login?: boolean;
+  main?: boolean;
+}
+
+const LeftSection: React.FC<HeaderProps> = ({ lecture }) => {
```

Comment:
여기도 이렇게 바꿀 수 있을 것 같네요
```suggestion
const LeftSection = ({ lecture }:HeaderProps) => {
```